### PR TITLE
Add golang version which use a technique named pre-computation

### PR DESCRIPTION
**Pull request by 199911**

### CODE REVIEW

**Comment by gaplo917** [comment]
Nice try :smile:  . It is a new technology about "future computation".

Before we compute, we know the answer

File: golang/ABCDEFGHPPP.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
+package main
+
+import "fmt"
+
+// I am using a programming skill call precomputation.
+// It has a very good performance
+const ans = `
+  34
+- 09
```
Comment (kennytm) [inline golang/ABCDEFGHPPP.go]: I don't this is a valid solution, a two-digit number does not start with `0`.

Comment (199911) [inline golang/ABCDEFGHPPP.go]: Fixed

**Comment by mingchuno** [comment]
@199911 can you change the dir name to sth like `golang-pre-compute` ?

**Comment by 199911** [comment]
@mingchuno Updated :+1:

**Pull request merged**